modular_skyrat/modules/modular_reagents/code/debuffs.dm

```diff
@@ -0,0 +1,56 @@
+//*
+//DEBUFFS - status effects that in some way hinder or remove the player's function.
+//*
+
+//MOBILE STASIS - same as regular stasis with the immobilization/hands-blocking traits taken off. Non-modular change has it set to also be checked by IS_IN_STASIS(mob).
+/datum/status_effect/grouped/stasis_mobile
+	id = "stasis"
+	duration = -1
+	tick_interval = 10
+	alert_type = /atom/movable/screen/alert/status_effect/stasis
+	var/last_dead_time
```

Comment:
Something like this is what we call documentation for variables. Every time you have a `var/something` in a class definition (so, not inside of a proc), you should be documenting it with a short sentence saying what it does.

Disclaimer: I don't know if this is what that variable means, but I wanted to give you an example.
```suggestion
	/// Time since the person last died.
	var/last_dead_time
```